src/components/NewHathorButton.js

```diff
@@ -78,7 +78,6 @@ const style = StyleSheet.create({
     borderColor: 'rgba(0, 0, 0, 0.5)',
   },
   touchable: {
-    padding: 16,
```

Comment:
What does this change?

Reply:
For some reason the button was padding itself into nothing so half the button would not be visible.